config/environments/development.rb

```diff
@@ -36,10 +36,9 @@
   # Use GoodJob adapter for Active Job.
   config.active_job.queue_adapter = :good_job
 
-  config.action_mailer.delivery_method = :notify
-  config.action_mailer.notify_settings = {
-    api_key: ENV.fetch("GOVUK_NOTIFY_API_KEY"),
-  }
+  # Use letter_opener to allow us to receive emails locally, without the need for Notify
```

Comment:
Your premise for adding this isn't correct.
You can see emails dispatched via Notify https://www.notifications.service.gov.uk
![screencapture-notifications-service-gov-uk-services-022acc23-c40a-4077-bbd6-fc98b2155534-notification-1609c6c1-51c0-406d-8f23-1a4000ad841a-2024-04-22-12_38_24](https://github.com/DFE-Digital/itt-mentor-services/assets/17162488/989f0083-08fe-484a-b098-38b656811825)

The current setup (to notify) is probably better, as it mimics what happens on production.